### PR TITLE
Add Bosnian

### PR DESCRIPTION
**Pull request by fal3n-4ngel**
## Adding a language Bosnian

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language Bosnian
- [x] I have incremented the language count in the readme.

### CODE REVIEW

**Comment by MrBrain295** [comment]
Why was it called Spanish?

**Comment by fal3n-4ngel** [comment]
I created one for spanish and then realised Espanol exists, so changed to Bosnian

**Comment by MrBrain295** [comment]
The Bosnian language is different from Spanish.

**Comment by fal3n-4ngel** [comment]
Dude , I changed the whole Script to Bosnian

**Comment by fal3n-4ngel** [comment]
I first added Spanish, Then later found that it exist as Espanol, so I changed the whole script to Bosnian, So it's Now Just a new lang Bosnian

**Comment by MrBrain295** [comment]
Whoops!

File: #/~Bosnian [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Prikažite "Hello World" i novi red.
+ Zaustavite program.
```
Comment (MrBrain295) [inline #/~Bosnian:2]: I don't speak Bosnian but "program" looks to me like English.

**Review by MrBrain295 — CHANGES_REQUESTED** [review]

File: #/~Bosnian [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Prikažite "Hello World" i novi red.
```
[the remark this answers is not in the document]
Comment (fal3n-4ngel) [reply #/~Bosnian:1]: Hello World Originally Transilates to 'Zdravo svijete', but not sure that it exactly means Hello World

[the remark this answers is not in the document]
Comment (MrBrain295) [reply #/~Bosnian]: What do you mean “originally translate to”?

**Comment by fal3n-4ngel** [comment]
I meant as every language is different, There is no oure translation, This is the closest match

**Review by Richienb — APPROVED** [review]

**Pull request merged**